tests/test_urban_nature_access.py

```diff
@@ -384,3 +389,72 @@ def test_split_greenspace(self):
 
         admin_vector = None
         admin_layer = None
+
```

Comment:
It might be worth a unit test for `_reclassify_and_multiply`.

Reply:
356f8e722